frontend/components/dialogs/ProjectHistoryDialog.vue

```diff
@@ -467,6 +498,10 @@ export default {
       const country = this.getCountryDetails(countryId)
       return country && country.name ? country.name : ''
     },
+    parseCountryOffice(countryId) {
+      const country = this.getCountryDetails(countryId)
+      return country && country.name ? country.name : ''
```

Comment:
why is the country office taking the countryId (same as in parseCountry), they are different objects

Reply:
Fixed